src/integration-test/kotlin/org/radarbase/redcap/managementportal/MpClientTest.kt

```diff
@@ -51,7 +51,7 @@ class MpClientTest {
             )
         Assert.assertNotNull(subject)
         Assert.assertEquals(
-            Integer.valueOf(IntegrationUtils.REDCAP_RECORD_ID_1),
+            IntegrationUtils.REDCAP_RECORD_ID_1.toString(),
             subject!!.externalId
         )
         Assert.assertEquals(
```

Comment:
perhaps we need another test to check that MP subject is updated correctly (using another GET on the subject and checking only required info has changed, esp if only attributes updated)